app/controllers/grade_entry_forms_controller.rb

```diff
@@ -246,6 +250,63 @@ def student_interface
     @student = current_user
   end
 
+  def get_mark_columns
+    grade_entry_form = GradeEntryForm.find(params[:id])
+    grade_entry_items_columns = grade_entry_form.grade_entry_items
+
+    c = grade_entry_items_columns.map do |column|
+      { 
+        :id => column.name,
+        :content => column.name + " (" + column.out_of.to_s + ")",
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.